core/include/webview/detail/engine_base.hh

```diff
@@ -315,6 +322,9 @@ protected:
       }
     }
   }
+  bool m_is_size_set = false;
```

Comment:
Please make these variables private, and make the `const` ones `static const` or `static constexpr`.

Reply:
Done in latest commit